src/pages/basic_data/institution_admin/FindPersonPanel.tsx

```diff
@@ -81,6 +88,26 @@ export const FindPersonPanel = () => {
                   />
                 )}
               </Field>
+
+              <Field name="user.nvi.verifiedAt.id">
+                {({ field, form }: FieldProps<NviVerification>) => (
```

Comment:
husk å gå gjennom PR og fjern sånne midlertidige ting, da går QA-prosessen mye raskere etterpå, siden man slipper å vente på ny review -> kommentar -> fix -> ny review

```suggestion
                {({ field }: FieldProps<NviVerification>) => (
```

Reply:
Skulle not satt den tilbake på draft igjen, beklager det.